images/linux/scripts/installers/rndgenerator.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  rndgenerator.sh
+##  Desc:  Install random number generator
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install haveged
+apt-get -y install haveged
+
+# Run tests to determine that the software installed as expected
+echo "Testing to make sure that script performed as expected, and basic scenarios work"
+for cmd in haveged; do
+    if ! command -v $cmd; then
+        echo "$cmd was not installed or not found on PATH"
+        exit 1
+    fi
+done
```

Comment:
Why do we need a loop here?

Reply:
Just for feature tools if someone decides to add rng-tools.